queue_services/entity-emailer/src/entity_emailer/email_tracker/tracker.py

```diff
@@ -0,0 +1,131 @@
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Functionality for tracking processing of messages in queue."""
+
+import json
+import nats
+
+from entity_emailer.email_processors import filing_notification
+from legal_api.utils import datetime
+from tracker.models import MessageProcessing
+from tracker.services import MessageProcessingService
+
+
+def get_message_id(raw_queue_msg: nats.aio.client.Msg):
```

Comment:
would like to just use the cloud event id to uniquely identify a payload but not all publishers use the cloud event format yet.  Needed to implement custom logic to uniquely identify a queue message id on a case to case basis.